paper/myst.yml

```diff
@@ -2,48 +2,58 @@
 version: 1
 project:
   id: ubcgif-2020-oldenburg-etal-exploration-geophysics-open-source
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  title: 3D electromagnetic modelling and inversion - a case for open source
+  description: Presenting arguments for adopting an open-source methodology for geophysics and provide some background about open-source software for electromagnetics.
   authors:
-    # - name: AUTHOR 1
-    #   email: EMAIL
-    #   corresponding: true
-    #   orcid: ORCID
-    #   affiliation:
-    #     id: ubc
-    #     name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
-    # - name: Lindsey J. Heagy
-    #   email: lheagy@eoas.ubc.ca
-    #   corresponding: true
-    #   orcid: 0000-0002-1551-5926
-    #   affiliation: ubc
-  # doi: DOI
-  # arxiv: ARXIV
-  # github: GITHUB-URL
-  # date: YYYY/MM/DD # date of publication
+    - name: Douglas W. Oldenburg
+      email: doug@eos.ubc.ca
+      corresponding: true
+      orcid: 0000-0002-4327-2124
+      affiliation:
+        id: ubc
+        name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
+    - name: Lindsey J. Heagy
+      email: lheagy@eoas.ubc.ca
+      orcid: 0000-0002-1551-5926
+      affiliation: ubc
+    - name: Seogi Kang
+      affiliation: Department of Geophysics, Stanford University, Stanford, California
+      email: sgkang09@stanford.edu
+      orcid: 0000-0002-9963-936X
+    - name: Rowan Cockett
+      affiliation: Curvenote Inc, Canmore, Alberta
+      email: rowan@curvenote.com
+      orcid: 0000-0002-7859-8394
+  doi: 10.1080/08123985.2019.1580118
+  arxiv: 1902.08245
+  github: https://github.com/simpeg-research/oldenburg-2018-AEM
+  date: 2019/03/24 # date of publication
```

Comment:
again, here we can stick with a 2020 publication date since that i consistent with the citation